2024.html

```diff
@@ -57,6 +57,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
               Use the name of the website for sorting if no author is present.
               Thank you!
             -->
+            <li>alifeee's <a href="https://alifeee.co.uk/" target="_blank">website</a>, <a href="https://blog.alifeee.co.uk/" target="_blank">blog</a>, and <a href="https://weeknotes.alifeee.co.uk/" target="_blank">weeknotes</a> (mostly <a href="https://gist.github.com/alifeee/75df204c2be3815806e1d4be47e54d08" target="_blank">manually</a>)</li>
```

Comment:
Hello @alifeee! Thanks for the submission.

I invite you to read this [article on the use of `target="_blank"`](https://css-tricks.com/use-target_blank/) and when to use this _attribute and value_ combo.

I don’t think Github will strip its CSS, so I invite you to publish an article on the topic on your own space instead and limit the number of links to content you manage yourself.

You can also check [previous years](https://css-naked-day.github.io/2022.html) for an example.

Cheers!

Reply:
Hi! Thanks for the review.

The `target="_blank"` article you link is very interesting, I had not come across such an opinion as-yet. I have removed them from this PR :)

I have also removed the GitHub link. I noticed after creating this PR the <https://css-naked-day.github.io/#tools> sub-heading of the main page. I was going to put a link to my script there, as I do think it is useful to have a tool to remove CSS from many HTML files. Let me know if you would rather me not create such a PR.

I have updated my change to `<li>alifeee's <a href="https://alifeee.co.uk/">website</a>, <a href="https://blog.alifeee.co.uk/">blog</a>, and <a href="https://weeknotes.alifeee.co.uk/">weeknotes</a></li>`. I understand that this is more sites than the usual (just one), so if you would rather fit with the style of the website, I can further limit this to link only to <https://alifeee.co.uk>. Do let me know :)

Thanks again for the review, and good work you do here!

Reply:
Good for me. :)

Feel free to add a PR about your script with a brief sentence. Thanks!